### ISSUE TITLE
DISCUSS: Lars Eggert: Section 2 looks like considerations, not instructions

### ISSUE THREAD

**Issue by alvestrand** [ietf-wg-mediaman/toplevel#9]
Lars Eggert's DISCUSS:

### Section 2, paragraph 1
```
  2.  Rules for the Registration of New Top-Level Media Types
```
These aren't really rules or a best current practice. At best, this is
an assembly of considerations around top-level media types one may
want to think about. I had expected much more practical prescriptive
rules that someone considering a new top-level media type can
follow. Could these be tightened up?

**Comment by alvestrand** [comment]
Proposed resolution: Rename section 22 to "Considerations".
Move or copy the MUST and SHOULD statements into the IANA considerations section, in a new section named "Rules for new top level type registration".

**Comment by alvestrand** [comment]
In -04, section 2.1 is still "Required criteria", but holds a number of criteria that are SHOULD.

Suggest renaming 2.1 to "Things that must be considered". Alternatively, move the non-MUST considerations down to 2.2.

**Comment by duerst** [comment]
Addressed with commit 996b4b43d3bb.
